rest_framework/utils/model_meta.py

```diff
@@ -96,10 +97,18 @@ def _get_forward_relationships(opts):
     """
     forward_relations = OrderedDict()
     for field in [field for field in opts.fields if field.serialize and field.rel]:
+        # For < django 1.6
+        if hasattr(field, 'to_fields'):
+            to_field = field.to_fields[0] if len(field.to_fields) else None
```

Comment:
I think theres a comment somewhere. Theres an array named to_fields which contains the to_field parameter as the first index. I don't know why they did this, maybe they planned to expand on that feature or had to remain backward compatible but this shows some code from the ForeignKey Class:
```python
# https://docs.djangoproject.com/en/1.7/_modules/django/db/models/fields/related/
class ForeignObject(RelatedField):
    requires_unique_target = True
    generate_reverse_relation = True
    related_accessor_class = ForeignRelatedObjectsDescriptor

    def __init__(self, to, from_fields, to_fields, swappable=True, **kwargs):
        self.from_fields = from_fields
        self.to_fields = to_fields

ForeignKey(ForeignObject):
during __init__:
    super(ForeignKey, self).__init__(to, ['self'], [to_field], **kwargs)
```

The documentation only refers to to_field and not to_fields.